tests/helpers.py

```diff
@@ -120,6 +120,7 @@ def reference_id_from_url(url):
     for i in range(len(TEST_POLYGON_ANNOTATIONS))
 ]
 
+TEST_INDEX_EMBEDDINGS_FILE = "https://scale-ml.s3.amazonaws.com/tmp/sasha/pytest_embeddings_payload.json"
```

Comment:
If this is stored in /tmp I believe it will get automatically deleted pretty soon, at which point the tests will fail. Unit test artifacts need to be in permanent storage.